UX_OVERHAUL_PLAN.md

```
@@ -0,0 +1,169 @@
+# SatTracker UX Overhaul Plan
+
+## Vision
```

markdownlint-cli2 (0.21.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)